trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/webapp/TmfConfigurationSourceTypeSerializer.java

```diff
@@ -48,6 +53,15 @@ public void serialize(ITmfConfigurationSourceType value, JsonGenerator gen, Seri
         if (!value.getConfigParamDescriptors().isEmpty()) {
             gen.writeObjectField("parameterDescriptors", value.getConfigParamDescriptors()); //$NON-NLS-1$
         }
+        File schemaFile = value.getSchemaFile();
```

Comment:
Not implemented

Reply:
https://github.com/eclipse-tracecompass/org.eclipse.tracecompass/pull/141